### PR TITLE
[UI] exam list page for user #18

### PR DESCRIPTION
**Pull request by edisonhello**
Preview

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,9 @@
     "bootstrap": "^4.4.1",
     "chart.js": "^2.9.3",
     "codemirror": "^5.52.2",
+    "dateformat": "^4.5.1",
     "highlight.js": "^9.18.1",
+    "humanize-duration": "^3.25.1",
```
Comment (Johnny850807) [inline package.json:32]: Isn't `moment.js` enough to support our requirement?

We have this in our dependencies already.

**Review by Johnny850807 — CHANGES_REQUESTED** [review]
Wow, you did a great job on fitting into my architecture!

Only one question on our need to include the new libraries.

**Comment by Chaoyuuu** [comment]
我只看UI的部分 😃 
ExamLists
可以參考Figma !!
image

File: package.json (hunk 1 above)
Comment (edisonhello) [reply package.json:32]: I didn't notice this. I'll try to migrate to it.

**Comment by edisonhello** [comment]
@Johnny850807 I've migrated `dateformat` to `moment` and remove `duration` part since we don't need duration now.
@Chaoyuuu "Exam List" words is already bold (with font-weight 700, as figma shows.) Should we make it bolder?

**Comment by edisonhello** [comment]
Also, I removed the `card` style. Preview now looks like this:
image

**Comment by Chaoyuuu** [comment]
Let's make "Exam List" bolder !!! 
Can you set the font-weight in 800 ?

**Review by Johnny850807 — APPROVED** [review]
LGTM. You did a great job, thanks.

**Comment by edisonhello** [comment]
@Chaoyuuu sure. This is how `font-weight: 800` looks like:
image

**Comment by Chaoyuuu** [comment]
I like this one better. We will use this style (`font-weight: 800`) in the future work. 
LGTM, thanks. 👍🏻

**Review by Chaoyuuu — APPROVED** [review]
LGTM

**Review by Johnny850807 — APPROVED** [review]
LGMT

**Review by Chaoyuuu — APPROVED** [review]
LGTM

**Comment by edisonhello** [comment]
This PR resolves #18, not #20. lol

**Pull request merged**